hiredis.h

```diff
@@ -234,12 +234,22 @@ redisContext *redisConnectUnixWithTimeout(const char *path, const struct timeval
 redisContext *redisConnectUnixNonBlock(const char *path);
 redisContext *redisConnectFd(redisFD fd);
 
+/**
+ *  Redis SSL mode
+ */
+enum redisSslMode {
```

Comment:
It may be nice to do a `typedef enum redisSslMode redisSslMode` here so you can then skip the enum when declaring these in other places.